applications/directory/src/hooks/reviewArtistProfile.js

```diff
@@ -5,7 +5,7 @@ const useReview = (token) => {
   const { httpClient } = useHttpClient();
 
   const { data: artist, error } = useSWR(
-    `/artists/token/${token}`,
+    `/artists/reviewToken/${token}`,
```

Comment:
RESTful URLs should be lowercase. Rename to `/artists/review-token/:token` here and in route definition in affected serverless.yml files (for both directory-api and artist-api).

Reply:
the routes for both apis is /artists/{field}/{value}. done